src/main/kotlin/org/rust/ide/intentions/RemoveParenthesesFromExprIntention.kt

```diff
@@ -16,7 +16,9 @@ class RemoveParenthesesFromExprIntention : RsElementBaseIntentionAction<RsParenE
     override fun getFamilyName(): String = text
 
     fun isAvailable(expr: RsParenExpr?): Boolean {
-        if (expr?.ancestorStrict<RsCondition>() == null && expr?.ancestorStrict<RsMatchExpr>() == null) return true
+        if (expr?.ancestorStrict<RsCondition>() == null &&
+            expr?.ancestorStrict<RsMatchExpr>() == null &&
+            expr?.ancestorStrict<RsForExpr>() == null) return true
```

Comment:
Consider use
```
PsiTreeUtil.getContextOfType(
        expr,
        false,
        RsCondition::class.java,
        RsMatchExpr::class.java,
        RsForExpr::class.java
    )
```
to avoid multiple tree traverse